v2/internal/identity/credential_provider.go

```diff
@@ -95,7 +98,7 @@ func NewCredentialProvider(
 // If no matching credential can be found, an error is returned.
 func (c *credentialProvider) GetCredential(ctx context.Context, obj genruntime.MetaObject) (*Credential, error) {
 	// Resource annotation
-	cred, err := c.getCredentialFromAnnotation(ctx, obj, common.PerResourceSecretAnnotation)
+	cred, err := c.getCredentialFromAnnotation(ctx, obj, annotations.PerResourceSecretAnnotation)
```

Comment:
Given the ~namespace~ package it's now in, can remove the suffix

```suggestion
	cred, err := c.getCredentialFromAnnotation(ctx, obj, annotations.PerResourceSecret)
```